src/mount/mount_unmount.rs

```diff
@@ -19,18 +19,18 @@ pub fn mount<Source: path::Arg, Target: path::Arg, Fs: path::Arg, Data: path::Ar
     target: Target,
     file_system_type: Fs,
     flags: MountFlags,
-    data: Data,
+    data: impl Into<Option<Data>>,
```

Comment:
I'm curious why this is `impl Into<Option<...>>` instead of just `Option<...>`.

Reply:
Just because it's annoying to write `Some(thing)` when you're primarily going to pass in `thing` all the time. I don't have a strong opinion there though.

Reply:
Ah, for some reason it didn't occur to me that you could do that. This sounds fine to me.